api.js

```diff
@@ -435,8 +435,8 @@ dbmigrate.prototype = {
   /**
    * Executes the default routine.
    */
-  run: function () {
-    load('run')(this.internals, this.config);
+  run: function (callback) {
+    load('run')(this.internals, this.config, callback);
```

Comment:
not a valid definition, everything needs to be a promise in the first place. callbacks are just second class citizens and just supported for backwards compatibility.